docs/user/howto/expandcharthypper.md

```diff
@@ -76,7 +76,7 @@ version: 8.5.1
 
 We added the `hypper.cattle.io/namespace: databases` to the annotation, so now when we install this chart with Hypper it will install into the proper namespace without any need to passing the value on the CLI:
 
-```shell
+```terminal
```

Comment:
This is confusing all the time I do not get the difference between `shell`, `terminal` and `console`. Can we use one thing consistent? (just to have the same format on every  terminal related snipped, independent of the md interpreter used)

Reply:
On https://github.com/rancher-sandbox/hypper/pull/131/commits/5deb37841b04b7ff06ac26c9df69b2133e41202f, I changed all the `shell` ones to `terminal` (because the shell ones syntax highlight shell builtins, vars, etc.

Comparing `terminal` and `console`, console seems to be better, so I will change them all to that. Example:

```shell
# this is shell highlighting
$ hypper install mysql/                 
Installing chart "mysql" in namespace "databases"…
Done! 👏 
```

```terminal
# this is terminal highlighting
$ hypper install mysql/                 
Installing chart "mysql" in namespace "databases"…
Done! 👏 
```

```console
# this is console highlighting
$ hypper install mysql/                 
Installing chart "mysql" in namespace "databases"…
Done! 👏 
```

Reply:
done in https://github.com/rancher-sandbox/hypper/pull/131/commits/11a1a71d1a07dd8e88488002afc980dddeae8c87.

Reply:
Thx for explaining :)